x-pack/platform/plugins/shared/onechat/public/application/utils/docs_paths.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const docsRoot = 'https://www.elastic.co/docs/solutions/search/elastic-agent-builder';
```

Comment:
doc paths are provided via another way, rather than you needing to hardcode the url for them.

see playground as an example:
- need to create a place for the onechat plugin to set doc_links (x-pack/solutions/search/plugins/search_playground/common/doc_links.ts)
- then the docLinks are set at plugin start (x-pack/solutions/search/plugins/search_playground/public/plugin.ts)